CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+* **WARNING:** nil values will now overwrite an existing value.  This change of behavior can be reverted via `config.merge_nil_values = false` in your Config initializer  ([#196](https://github.com/railsconfig/config/pull/196))
```

Comment:
I retained the format of the rest of the file here and didn't fix the line length cop.

Reply:
👍